standalone/rewrite_config.go

```diff
@@ -32,12 +33,19 @@ func main() {
 
 	//separate the lines in each node log
 	for i := 0; i < (len(node_logs) - 1); i++ {
+		var swarm_port string
 		var api_port string
 		var remote_api_port string
 
 		//separate the lines on newline character
 		lines := strings.Split(node_logs[i], "\n")
 		for j := 0; j < (len(lines)); j++ {
+			if strings.HasPrefix(lines[j], "Swarm listening on /ip4/") {
+				//split to find port value
+				parts := strings.Split(lines[j], "/")
+				swarm_port = parts[(len(parts))-1]
```

Comment:
here if len(parts) is 0, it might crash. Better add check if len(parts) is 0, then add error log something.

Reply:
ok good idea.